src/bindings/python/fluxacct/accounting/job_usage_calculation.py

```diff
@@ -120,7 +120,7 @@ def apply_decay_factor(decay, acct_conn, user=None, bank=None):
     # update job_usage_factor_table with new values, starting with period-2;
     # the last usage factor in the table will get discarded after the update
     period = 1
-    for usage_factor in usg_past_decay[1 : len(usg_past_decay) - 1]:
+    for usage_factor in usg_past_decay[0 : len(usg_past_decay) - 1]:
```

Comment:
Is the comment above here about "starting with period-2" still correct after this change?

Also I think the line could be simplified to `usg_past_decay[0:-1]` since negative indices are counted from the end:

```
>>> x
[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
>>> x[0:len(x)-1]
[0, 1, 2, 3, 4, 5, 6, 7, 8]
>>> x[0:-1]
```